### PR TITLE
chore: Specify Default CMM Initialization

### PR DESCRIPTION
**Pull request by lavaleri**
_Issue #, if available:_ Add specification for Default CMM initialization

_Description of changes:_

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: framework/default-cmm.md [hunk 1]
```diff
@@ -43,6 +43,18 @@ It is used by default to wrap the key provider.
...
+
+On defualt CMM initialization,
+the caller MUST provide the following value:
+
+- [Keyring](#keyring)
+
+### Keyring
+
+The [keyring](keyring-interface.md) this CMM uses to
+[retrieve encryption materials](#get-encryption-materials) or [decrypt materials](#decrypt-materials).
```
Comment (alex-chew) [inline framework/default-cmm.md:56]:
```suggestion
[get encryption materials](#get-encryption-materials) or [decrypt materials](#decrypt-materials).
```

File: framework/default-cmm.md [hunk 2]
```diff
@@ -101,11 +115,22 @@ using master key providers.
 This is a legacy specification.
 Master key providers SHOULD NOT be included in any additional implementations.
 
+### Initialization
+
+In place of requiring the caller to provide a [keyring](#keyring-interface.md)
+on initialization:
+
+On defualt CMM initialization,
```
Comment (seebees) [inline framework/default-cmm.md:123]:
```suggestion
On default CMM initialization,
```

File: framework/default-cmm.md [hunk 3]
```diff
@@ -43,6 +43,18 @@ It is used by default to wrap the key provider.
 The key words "MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL NOT", "SHOULD", "SHOULD NOT", "RECOMMENDED", "MAY", and "OPTIONAL"
 in this document are to be interpreted as described in [RFC 2119](https://tools.ietf.org/html/rfc2119).
 
+## Initialization
+
+On defualt CMM initialization,
```
Comment (seebees) [inline framework/default-cmm.md:48]:
```suggestion
On default CMM initialization,
```

File: framework/default-cmm.md [hunk 4]
```diff
@@ -101,11 +115,22 @@ using master key providers.
 This is a legacy specification.
 Master key providers SHOULD NOT be included in any additional implementations.
 
+### Initialization
+
+In place of requiring the caller to provide a [keyring](#keyring-interface.md)
```
Comment (seebees) [inline framework/default-cmm.md:120]: Is this the right link?

**Comment by lavaleri** [comment]
We list OnEncrypt and OnDecrypt under the Operation header in the keyring specs, so updated to use that wording.

File: framework/default-cmm.md (hunk 4 above)
Comment (lavaleri) [reply framework/default-cmm.md]: nope. fixed.

**Pull request merged**